baremetal/metal3data_manager.go

```diff
@@ -347,6 +357,7 @@ func (m *DataManager) getAddressesFromPool(ctx context.Context,
 	}
 	if m3dt.Spec.NetworkData != nil {
 		for _, network := range m3dt.Spec.NetworkData.Networks.IPv4 {
+			m.Log.Info("Fetch network data from IP Pool for IPv4", "Pool Name", network.IPAddressFromIPPool)
```

Comment:
Let's keep this unified everywhere:
```suggestion
			m.Log.Info("Fetch network data from IPPool for IPv4", "Pool Name", network.IPAddressFromIPPool)
```